lib/ecto/adapters/sql.ex

```diff
@@ -663,8 +665,6 @@ defmodule Ecto.Adapters.SQL do
     for sql <- List.wrap(sqls) do
       query!(meta, sql, [], opts)
     end
-
-    {:ok, []}
```

Comment:
As mentioned, let's continue returning `{:ok, ...}`. :) This is important in case we want to return `:error` or something similar in the future.